### DISCUSSION TITLE
看不見蚊子？黑科技幫你開外掛找蚊子

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#227] [General]
https://technews.tw/2022/01/11/bzigo-infra-red-ai-mosquito-tracking-device-unveiled-in-ces-2022/
隨著疫情膠著讓許多人待在家的時間越來越多，蚊蟲困擾令人難以忍受。美國廠商 Bzigo 於今年 CES 2022 線上展示間推出雷射蚊蟲追蹤裝置，透過紅點標定蚊蟲位置，讓電蚊拍不再揮空。

**Comment by Yunhans** [comment]
在裝上bb槍就連打蚊子都不需要了
